client/components/common/tce-core/Discussion/TextEditor.vue

```diff
@@ -5,7 +5,7 @@
     </div>
     <v-textarea
       v-else
-      @change="$emit('input', $event)"
+      @input="$emit('input', $event)"
```

Comment:
Think about proxying events and attributes and using `v-model.trim` in the parent